pom.xml

```diff
@@ -0,0 +1,175 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>xxx</groupId>
+    <artifactId>yyy</artifactId>
+    <!-- do not change this version. Done by the workflow before pushing the artifact to Maven Central -->
+    <version>1.0.0-SNAPSHOT</version>
+    <name>zzz</name>
+    <description>aaa</description>
+    <url>https://github.com/Hapag-Lloyd/bbb</url>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
Not sure if we want to support Java 8 still by default.. Is that a requirement? If so, why?

Reply:
Thought about the https://github.com/Hapag-Lloyd/json-logic-rule-maker project. But 21 should be the default.